prijateli_tree/app/routers/games.py

```diff
@@ -244,6 +244,9 @@ def view_round(
     Function that returns the current round
     """
     game, player = get_game_and_player(game_id, player_id, db)
+    session_player = get_session_player_from_player(player, db)
+    player_name = f"{player.user.first_name} {player.user.last_name}"
+    player_score = session_player.points
 
```

Comment:
You're repeating this code  a lot. Write a util function to get the information need for the header.